.travis.yml

```diff
@@ -80,34 +80,7 @@ before_script:
       "
   - ./configure $CONFIGURE_FLAGS
 script:
-  - . tests/build-log-cflags-propagation.sh;
-    if [ "$CC" = "gcc" ]; then
-      export DISTCHECK_CONFIGURE_FLAGS="$CONFIGURE_FLAGS";
-      exec_prop_check "make distcheck -j 3 V=1 --keep-going" ||
-      {
-        S=$?;
-        make V=1 distcheck;
-        find . -name test-suite.log | xargs cat;
-        return $S;
-      };
-      find . -name test-suite.log | xargs cat;
-    else
-      make --keep-going -j $(nproc);
-      S=$?;
-      if [ "$S" = "0" ]; then
-        make install
-        . scripts/get-libjvm-path.sh || return $?;
-        export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$JNI_LIBDIR";
-        make func-test V=1;
-        make pytest-self-check;
-        make pytest-check;
-      elif [ "$S" = "42" ]; then
-        return $S;
-      else
-        make V=1 install;
-        return $S;
-      fi;
-    fi
+  - tests/travis-common-script.sh
```

Comment:
You have to export `CONFIGURE_FLAGS` before executing a child process that uses it.

Reply:
+1, Thanks for the review and a comment.